tests/config/database.php

```diff
@@ -1,35 +1,22 @@
 <?php
 
-$mongoHost = env('MONGO_HOST', 'mongodb');
-$mongoPort = env('MONGO_PORT') ? (int) env('MONGO_PORT') : 27017;
-$mysqlPort = env('MYSQL_PORT') ? (int) env('MYSQL_PORT') : 3306;
-
 return [
-
     'connections' => [
-
         'mongodb' => [
             'name' => 'mongodb',
             'driver' => 'mongodb',
-            'host' => $mongoHost,
+            'dsn' => env('MONGODB_URI', 'mongodb://127.0.0.1/'),
             'database' => env('MONGO_DATABASE', 'unittest'),
-        ],
-
-        'dsn_mongodb' => [
-            'driver' => 'mongodb',
-            'dsn' => "mongodb://$mongoHost:$mongoPort",
-            'database' => env('MONGO_DATABASE', 'unittest'),
-        ],
-
-        'dsn_mongodb_db' => [
-            'driver' => 'mongodb',
-            'dsn' => "mongodb://$mongoHost:$mongoPort/".env('MONGO_DATABASE', 'unittest'),
+            'options' => [
+                'connectTimeoutMS'         => 100,
+                'serverSelectionTimeoutMS' => 250,
+            ],
         ],
 
         'mysql' => [
             'driver' => 'mysql',
             'host' => env('MYSQL_HOST', 'mysql'),
-            'port' => $mysqlPort,
+            'port' => env('MYSQL_PORT') ? (int) env('MYSQL_PORT') : 3306,
```

Comment:
This change seems unrelated to the PR.

Reply:
Probably cleaner to do that, and it shouldn't conflict with anything in this PR since the lines aren't adjacent.